### DISCUSSION TITLE
How can I add this library to dev-c++?

### DISCUSSION THREAD

**Discussion by ghost** [ArthurSonzogni/FTXUI#375] [Q&A] [answered]
I couldn't find any tutorials explaining how to add this specific library to it. If you know how to do it please let me know; It would be really helpful. Thanks :D!

**Comment by ArthurSonzogni** [comment, marked as answer]
Hello Andrew-Hernandez!

I don't know what is dev-c++. It looks quite old and probably do not really support CMake, which is what most C++ project use these days.

I suspect you might be using Windows? I am not really familiar, but I believe people are mostly Visual studio, visual studio code and QT creator.

**Reply by ghost** [reply]
Is there any documentation for installing it in the visual studio application?

**Reply by ArthurSonzogni** [reply]
I don't use an IDE, but maybe this documentation can help you?
https://docs.microsoft.com/en-us/cpp/build/cmake-projects-in-visual-studio?view=msvc-170

Maybe you can download:
https://github.com/ArthurSonzogni/ftxui-starter
and try opening it inside this IDE?
